src/main/java/Ryan.java

```diff
@@ -0,0 +1,58 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Ryan {
+    public static void main(String[] args) {
+        boolean exit = false;
+        Scanner scanner = new Scanner(System.in);
+        ArrayList<Task> tasks = new ArrayList<>();
```

Comment:
Very good in ensuring ArrayList is defied as containing tasks types for type safety.